biz.aQute.bndlib.tests/test/test/servicecapability/ServiceCapabilityTest.java

```diff
@@ -0,0 +1,98 @@
+package test.servicecapability;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+import org.osgi.namespace.service.ServiceNamespace;
+
+import aQute.bnd.annotation.service.ServiceCapability;
+import aQute.bnd.header.Attrs;
+import aQute.bnd.header.Parameters;
+import aQute.bnd.osgi.Builder;
+import aQute.bnd.osgi.Domain;
+import aQute.lib.io.IO;
+import test.servicecapability.misc.Foo;
+import test.servicecapability.misc.Foo2;
+import test.servicecapability.misc.Foo3;
+import test.servicecapability.misc2.Bar;
+
+@ServiceCapability(value = ServiceCapabilityTest.class)
+@ServiceCapability(value = Foo.class, attribute = "test=test")
+@ServiceCapability(value = Foo2.class, uses = Bar.class)
+@ServiceCapability(value = Foo3.class, uses = {
+	Bar.class, ServiceCapabilityTest.class
+})
+public class ServiceCapabilityTest {
+
+
+	@Test
+	public void testServiceCapability() throws Exception {
+		Attrs attrs = assertCapability(ServiceCapabilityTest.class);
+		assertEquals("test.servicecapability", attrs.get("uses:"));
+	}
+
+	@Test
+	public void testServiceCapabilityBar() throws Exception {
+		Attrs attrs = assertCapability(Bar.class);
+		assertEquals("test.servicecapability.misc2", attrs.get("uses:"));
+	}
+
+	@Test
+	public void testServiceCapabilityFoo() throws Exception {
+		Attrs attrs = assertCapability(Foo.class);
+		assertEquals("test.servicecapability.misc", attrs.get("uses:"));
+		assertEquals("test", attrs.get("test"));
+	}
+
+	@Test
+	public void testServiceCapabilityFoo2() throws Exception {
+		Attrs attrs = assertCapability(Foo2.class);
+		assertEquals("test.servicecapability.misc2", attrs.get("uses:"));
+	}
+
+	@Test
+	public void testServiceCapabilityFoo3() throws Exception {
+		Attrs attrs = assertCapability(Foo3.class);
+		assertTrue(attrs.get("uses:")
+			.contains("test.servicecapability.misc2"));
```

Comment:
```suggestion
		assertThat(attrs).hasEntrySatisfying("uses:", v -> v.contains("test.servicecapability.misc2"));
```

Another assertj example.

Reply:
done